### PR TITLE
fix: focus outline on header bottom-bar

### PR DESCRIPTION
**Pull request by felix-ico**
this should fix https://github.com/telekom/scale/issues/2203

### CODE REVIEW

File: packages/components/src/components/telekom/telekom-mobile-menu-item/telekom-mobile-menu-item.css [hunk 1]
```diff
@@ -99,4 +99,5 @@
 :host::part(icon-right-container) {
   display: flex;
   align-items: center;
+  position: absolute;
```
Comment (acstll) [inline packages/components/src/components/telekom/telekom-mobile-menu-item/telekom-mobile-menu-item.css:102]: I wonder why is this needed, I couldn't see the effect 👀

File: packages/components/src/components/telekom/telekom-nav-list/telekom-nav-list.css [hunk 2]
```diff
@@ -104,14 +104,43 @@
...
+  );
 }
-.scale-telekom-nav-list[alignment='center'] > * {
-  margin-inline: calc(0.5 * var(--_spacing-x-slotted));
+
+.scale-telekom-nav-list[variant='main-nav']
+  > .scale-telekom-nav-item
+  button::after,
+.scale-telekom-nav-list[variant='main-nav'] > .scale-telekom-nav-item a::after {
+  margin-left: var(--telekom-spacing-composition-space-06);
```
Comment (acstll) [inline packages/components/src/components/telekom/telekom-nav-list/telekom-nav-list.css:137]: Why no `margin-inline-start`? 😛

**Review by acstll — APPROVED** [review]
Left a couple of comments but I think it's good to merge anyways 💪

File: packages/components/src/components/telekom/telekom-mobile-menu-item/telekom-mobile-menu-item.css (hunk 1 above)
Comment (felix-ico) [reply packages/components/src/components/telekom/telekom-mobile-menu-item/telekom-mobile-menu-item.css:102]: Screenshot 2023-12-20 at 15 03 16

without it the focus outline doesn't include the icon...
